tree/ntuple/v7/src/RField.cxx

```diff
@@ -2947,3 +2949,66 @@ void ROOT::Experimental::RCollectionField::CommitClusterImpl()
 {
    *fCollectionNTuple->GetOffsetPtr() = 0;
 }
+
+//------------------------------------------------------------------------------
+
+const ROOT::Experimental::Detail::RFieldBase::RColumnRepresentations &
+ROOT::Experimental::RField<ROOT::Experimental::RNTupleBLOB>::GetColumnRepresentations() const
```

Comment:
For the other template specializations, this method and the ones below are implemented in the header. Should the other implementations also be moved?

Reply:
Not sure I follow. The `GetColumnRepresentations()` method is implemented in the compilation unit for all template specializations, isn't it?

Reply:
Ah right I got confused, sorry!